File: Examples/VisualBasic/Working-with-Text/InsertTextShape.vb
Imports Aspose.Diagram
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System
Public Class InsertTextShape
    Public Shared Sub Run()
        ' ExStart:InsertTextShape
        ' The path to the documents directory.
        Dim dataDir As String = RunExamples.GetDataDir_ShapeText()

        ' Create a new diagram
        Dim diagram As New Diagram()
        ' Set parameters and add text to a Visio page
        Dim PinX As Double = 1, PinY As Double = 1, Width As Double = 1, Height As Double = 1
        diagram.Pages(0).AddText(PinX, PinY, Width, Height, "Test text")
        ' Save diagram 
        diagram.Save(dataDir & Convert.ToString("InsertTextShape_out.vsdx"), SaveFileFormat.VSDX)
        ' ExEnd:InsertTextShape
    End Sub
End Class

File: Examples/CSharp/Working-with-Text/InsertTextShape.cs
using Aspose.Diagram;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Aspose.Diagram.Examples.CSharp.Working_with_Text
{
    public class InsertTextShape
    {
        public static void Run()
        {
            // ExStart:InsertTextShape
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_ShapeText();

            // Create a new diagram
            Diagram diagram = new Diagram();
            // Set parameters and add text to a Visio page
            double PinX = 1, PinY = 1, Width = 1, Height = 1;                  
            diagram.Pages[0].AddText(PinX, PinY, Width, Height, "Test text");
            // Save diagram 
            diagram.Save(dataDir + "InsertTextShape_out.vsdx", SaveFileFormat.VSDX);
            // ExEnd:InsertTextShape
        }
    }
}
